index.rst

```diff
@@ -10,20 +10,849 @@
+
+.. _Deliverable 2:
+
+Deliverable 2: Rapid Analysis Calculated Metrics
+------------------------------------------------
+
+.. note::
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+
+   2. (`SITCOM-180`_, `SITCOM-173`_) Define which metrics, analyses and artifacts must be calculated and on what timescale they must be evaluated and reported to support commissioning/operations.
+
+      This is to evaluate if a "rapid processing" of data is required, what specific calculations are required.
+      This list should include the relevant camera specific calculations (which are currently performed by the EO testing data reduction).
+      This is expected to inform the answer to the next charge task.
+
+
+It is important to note that the charge question above refers to "rapid processing."
+We intentionally avoid the use of this term and have adopted the phrasing, "Rapid Analysis" instead.
+This is to avoid any potential confusion with Prompt Processing, which is a discussed previously in the `Rapid Analysis`_ section.
+
+Numerous calculations are required to evaluate both camera and control system health and performance on rapid timescales.
+The data products discussed in this section are limited to scalars and/or arrays and do *not* include diagnostic plots and/or figures (visualization use cases are discussed separately).
+The large majority of data products needed on rapid timescales are produced as part of the Science Pipelines single-frame-processing (SFP) framework.
+A small number of additional values are also required, but can be quickly derived from the SFP results.
+The calculated values by Rapid Analysis are not to produce data products that are fundamental to continuing observations (`FAFF-REQ-0053`_), however, it is expected that observatory functionality will be reduced if an outage were to occur.
+This implies that the Rapid Analysis payload is not required to run at the summit, although it would be preferable to do so.
+The output from the Rapid Analysis will need to go into a database.
+Details of this are database are discussed in `Deliverable 3`_.
+The output will also have to be made available to the control network such that observers can be alerted if calculated metrics are producing results that are deemed worrisome.
+The original framework to perform this duty is the Telemetry Interface, described in :ref:`LSE-72`, which is designed to feed metrics from Prompt Processing pipelines running at the USDF back to the summit.
+The :ref:`LSE-72` document is out-of-date, however, either this or an analogous framework is required to perform the same purpose, such that the Watcher CSC can monitor for troubling events and alert the observer.
```

Comment:
"... can monitor for troubling events and alert the observer." -> "... can monitor and alert observers of troubling events."